src/api/app/controllers/webui/badge_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Webui::BadgeController < Webui::WebuiController
+  before_action :set_project, only: [:show]
+  before_action :require_package, only: [:show]
```

Comment:
```suggestion
  before_action :set_project
  before_action :require_package
```
There is no need to define `only: [:show]` when there is only one action in the controller.